src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] It will be great to have the values of each action type follow the duck's pattern. This is a major requirement for this project to be approved. What it means is that the values of each type should have something like `app/reducer/ACTION_TYPE`. To learn more about this kindly follow this [link](https://github.com/erikras/ducks-modular-redux)

Reply:
Done.

Reply:
This issue is well resolved! ✔️